app/src/main/java/ch/epfl/skysync/navigation/MainGraph.kt

```diff
@@ -17,9 +17,10 @@ fun MainGraph(
     user: FirebaseUser?
 ) {
   NavHost(
-      navController = navHostController,
-      startDestination = if (user == null) Route.LOGIN else Route.MAIN) {
-        homeGraph(navHostController)
+      navController = navHostController, startDestination = Route.MAIN
+      // startDestination = if (user == null) Route.LOGIN else Route.MAIN
```

Comment:
Shouldn't you remove the startDestination of line 20 and uncomment line 21

Reply:
the Login still has some problems. It will be switched back once it gets fixed.